quickcheck/Test/Tasty/QuickCheck.hs

```diff
@@ -169,34 +179,41 @@ instance IsOption QuickCheckMaxShrinks where
 -- but may be used by others.
 --
 -- @since 0.9.1
-optionSetToArgs :: OptionSet -> IO (Int, QC.Args)
+optionSetToArgs :: OptionSet -> IO QC.Args
```

Comment:
Notice the API breaking change. `optionSetToArgs` is exported.

Reply:
I don't see a reason for breakage though. You can still return `(Int, QC.Args)`.

Reply:
Indeed. I reverted this.